edkahara/iReporter
# [finishes #162422784] create v2 signup endpoint

__What does this PR do?__
Create a sign up endpoint for v2 of iReporter.

__Description of tasks to be completed.__
Create an api endpoint that facilitates the creation of a new user. Test the api endpoint.

__How should it be manually tested?__
Clone the repository. Check out the ft-create-v2-signup-endpoint-#162422784 branch. Start the environment using env\Scripts\activate and use env.bat to set environment variables. Run flask run on the terminal and go to Postman. To perform the unit tests run nosetests.

__Relevant pivotal tracker stories.__
#162422784
